default/stringtables/en.txt

```diff
@@ -204,13 +204,13 @@ NEW_ASTEROIDS_SUFFIX
 [[PT_ASTEROIDS]]
 
 EMPTY_SPACE
-Deep Space
+''' (empty)'''
```

Comment:
This is possibly misleading... there could be ships in the system, and possibly planets even with no star. What is shown in those cases?

Reply:
right, the same. so - change to " (no star)" instead?

Reply:
wait a moment .. no, normal pre-change behaviour should apply, so only the really empty systems that show no name on the map will get the suffix and it will not show up in starless systems that were not called Deep Space before. still, might be fleets there so still somewhat misleading and still i'll put the no-star-suffix in next. and still not hewn in stone.

Reply:
did not find a starless planet right now.

![empty systems 'no star' no star](https://user-images.githubusercontent.com/37298400/156823483-8f11ffa1-a1d2-47b3-b150-878e0a8722a4.png)